packages/zoe/test/unitTests/contracts/test-automaticRefund.js

```diff
@@ -68,33 +62,33 @@ test('zoe - simplest automaticRefund', async t => {
 test('zoe - automaticRefund same issuer', async t => {
   try {
     // Setup zoe and mints
-    const { issuers, moola } = setup();
+    const { issuers, mints, moola } = setup();
     const [moolaIssuer] = issuers;
+    const [moolaMint] = mints;
     const zoe = makeZoe({ require });
     // Pack the contract.
     const { source, moduleFormat } = await bundleSource(automaticRefundRoot);
     const installationHandle = zoe.install(source, moduleFormat);
 
+    // Setup Alice
+    const aliceMoolaPayment = moolaMint.mintPayment(moola(9));
```

Comment:
Why wasn't this necessary before?

Reply:
Oh, what do you mean? This view doesn't show me the full diff, and I can't find this comment elsewhere.

Reply:
AliceMoolaPayment didn't exist before. Here it's being passed into alicePayments below (line 85). In the previous version alicePayments was `[undefined, undefined]`.  Why did this need to change?

<img width="1479" alt="auto-refund-moolaPmnt" src="https://user-images.githubusercontent.com/13372636/76122371-bf7de780-5faa-11ea-8cca-f650368ba732.png">

Reply:
Alice didn't try to offer anything before